ada-feeding.rosinstall

```diff
@@ -0,0 +1,17 @@
+- git: {local-name: ada_demos, uri: 'https://github.com/personalrobotics/ada_demos.git'}
+- git: {local-name: ada_description, uri: 'https://github.com/personalrobotics/ada_description.git'}
+- git: {local-name: ada_launch, uri: 'https://github.com/personalrobotics/ada_launch.git'}
+- git: {local-name: aikido, uri: 'https://github.com/personalrobotics/aikido.git'}
+- git: {local-name: face_detection, uri: 'https://github.com/personalrobotics/face_detection.git'}
+- git: {local-name: food_detector, uri: 'https://github.com/personalrobotics/food_detector.git'}
+- git: {local-name: forque_sensor_hardware, uri: 'https://github.com/personalrobotics/forque_sensor_hardware.git'}
+- git: {local-name: jaco_hardware, uri: 'https://github.com/personalrobotics/jaco_hardware.git'}
+- git: {local-name: libada, uri: 'https://github.com/personalrobotics/libada.git'}
+- git: {local-name: net-ft-ros, uri: 'https://github.com/personalrobotics/net-ft-ros.git'}
+- git: {local-name: pose_estimators, uri: 'https://github.com/personalrobotics/pose_estimators.git'}
+- git: {local-name: pr_assets, uri: 'https://github.com/personalrobotics/pr_assets.git'}
+- git: {local-name: pr_control_msgs, uri: 'https://github.com/personalrobotics/pr_control_msgs.git'}
+- git: {local-name: pr_hardware_interfaces, uri: 'https://github.com/personalrobotics/pr_hardware_interfaces.git'}
+- git: {local-name: pr_ros_controllers, uri: 'https://github.com/personalrobotics/pr_ros_controllers.git'}
+- git: {local-name: pr_tsr, uri: 'https://github.com/personalrobotics/pr_tsr.git'}
+- git: {local-name: rewd_controllers, uri: 'https://github.com/personalrobotics/rewd_controllers.git'}
```

Comment:
As mentioned above, we need `bite_selection_package` and `pytorch_retinanet` as well.